File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.kt
class Solution {
    fun solution(num_list: IntArray): IntArray {
        var answer: IntArray = intArrayOf()
        val last = num_list[num_list.size - 1]
        val secondLast = num_list[num_list.size - 2]

        return if (last > secondLast) {
            num_list + (last - secondLast)
        } else {
            num_list + (last * 2)
        }
        
        answer = num_list
        return answer
    }
}

File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.py
def solution(num_list):
    answer = []
    answer = num_list
    
    if (num_list[-1] > num_list[-2]): # 마지막 원소가 더 큰 경우
        answer.append(num_list[-1] - num_list[-2])
    else:
        answer.append(num_list[-1]*2)
    return answer